docs/api/instruments/ni/index.rst

```diff
@@ -0,0 +1,12 @@
+.. module:: pymeasure.instruments.ni
+
+####################
+National Instruments
+####################
+
+This section contains specific documentation on the National Instruments instruments that are implemented. If you are interested in an instrument not included, please consider :doc:`adding the instrument </dev/adding_instruments>`.
+
+.. toctree::
+   :maxdepth: 2
+
+   virtualbench
```

Comment:
We should also include the DAQmx docs here...but this can be considered out of scope for this PR.

Reply:
the DAQmx code does merely contain docstrings so far, so I don't add it to the index file now. As I don't have such an instrument I can't help on the documentation. A separate PR is good idea!